.travis.yml

```diff
@@ -22,8 +13,35 @@ jobs:
     - stage: test
       script:
         - |
-          if [[ "$TRAVIS_PYTHON_VERSION" != "2.6" ]] ; then make test-readme; fi
+          make test-readme
         - make ci
+      python: '2.7'
+    - stage: test
+      script:
+        - |
+          make test-readme
+        - make ci
+      python: '3.4'
+    - stage: test
+      script:
+        - |
+          make test-readme
+        - make ci
+      python: '3.5'
+    - stage: test
+      script:
+        - |
+          make test-readme
+        - make ci
+      python: '3.6'
+    - stage: test
+      script:
+        - |
+          make test-readme
+        - make ci
+      python: '3.7'
+      dist: xenial
+      sudo: true     
```

Comment:
No, it does not. This is the only way to use xenial on Travis is to use sudo on this job alone. And xenial is the only target that has true Python 3.7. So it's not ideal (because enabling sudo is *slow*) but it's the best we have right now